### PR TITLE
Add definitions for gadgets, tools and bots to Glossary

### PR DESCRIPTION
**Pull request by tzhelyazkova**
Bug: [T272970](https://phabricator.wikimedia.org/T272970)

### CODE REVIEW

File: systems/overview/12-Glossary.md [hunk 1]
```diff
@@ -55,3 +55,15 @@ MediaWiki Core refers to all files that are part of the main MediaWiki package.
 ## [Special Page](https://www.mediawiki.org/wiki/Manual:Special_pages)
 
 Special pages are dynamically created MediaWiki pages, which perform a specific function, such as providing a list of pages, showing statistics or creating a form for user submitted feedback. They are located in their own namespace (_Special:_) and are not editable directly like other pages.  Developers can create new special pages. They will generally show up in the list of all special pages at [Special:SpecialPages](https://www.wikidata.org/wiki/Special:SpecialPages).
+
+## Gadgets
+
+Gadgets are [tools](#tools) which can be enabled from you Wikidata's account preferences.
```
Comment (jakobw) [inline systems/overview/12-Glossary.md:61]: Hm, I tried to find a link to an "official" description of what Gadgets are, but couldn't find any. Maybe we can add a sentence to mention that they're enabled via https://www.mediawiki.org/wiki/Extension:Gadgets

**Review by jakobw — COMMENTED** [review]
Looks good.

Would it make sense to add third party applications to this list?

File: systems/overview/12-Glossary.md (hunk 1 above)
Comment (tzhelyazkova) [reply systems/overview/12-Glossary.md:61]: Yep, me neither, so I came up with this sentence. I'm not even 100% sure that gadgets and tools are synonyms in Wikidata's context.
Will add the link you mentoned.

**Comment by manicki** [comment]
My own interpretation of those terms that I have developed for myself (not based on any docs as I have never pointed to any, so I don't have references) was that a gadget is basically an arbitrary JS code run on wiki pages - hopefully extending the functionality of Mediawiki/Wikibase UI.
Tools are any kind of software used together with MW/Wikibase but running outside of wikimedia sites (generally as described on toolforge). Tools are generally human-operated (e.g. need to go to a tool UI, click stuff)
Bots are programs that do edits to Wikipedia/Wikidata data without human interaction (although they are of course instructed by human what/how to do stuff)

No idea how those are accurate, but they are somehow not fully in line with what is described in this PR for now, so I would be personally curious to get an "official" right interpretations of these

File: systems/overview/12-Glossary.md (hunk 1 above)
Comment (jakobw) [reply systems/overview/12-Glossary.md:61]: I like the distinction we've seen in the Mattermost chat and made by @manicki (thanks!), i.e. Gadgets = "tools" that run on the wiki, tools = external applications or scripts.

We can also steal some of the description from https://www.mediawiki.org/wiki/Extension:Gadgets :
```suggestion
[Gadgets](https://www.mediawiki.org/wiki/Extension:Gadgets) are made up of JavaScript and/or CSS Snippets located on pages in the MediaWiki namespace. These snippets add functionality to the wiki itself and can be enabled and disabled via preferences.
```

**Review by Silvan-WMDE — APPROVED** [review]

**Pull request merged**